src/trg-client.c

```diff
@@ -468,8 +468,7 @@ void trg_client_configunlock(TrgClient * tc)
 void trg_response_free(trg_response * response)
 {
 	if (response) {
-		if (response->obj)
-			json_object_unref(response->obj);
+                g_clear_pointer(&response->obj, json_object_unref);
```

Comment:
Whitespace seems off.

Reply:
I just saw this, I think this is tabs vs spaces things, I'll switch to tabs for this file.